camdecmps/procedures/refresh_emission_view_noxmasscems.sql

```diff
@@ -39,59 +45,130 @@ BEGIN
 		CALC_NOX_MASS,
 		ERROR_CODES
 	)
-  SELECT DISTINCT
-		HOD.MON_PLAN_ID, 
-		HOD.MON_LOC_ID, 
-		HOD.RPT_PERIOD_ID, 
+  SELECT
+		hod.MON_PLAN_ID, 
+		hod.MON_LOC_ID, 
+		hod.RPT_PERIOD_ID, 
 		camdecmps.format_date_hour(hod.BEGIN_DATE, hod.BEGIN_HOUR, null) AS DATE_HOUR,
-		HOD.OP_TIME, 
-		HOD.HR_LOAD AS UNIT_LOAD, 
-		HOD.LOAD_UOM_CD AS LOAD_UOM, 
-		NOXC_MHV.UNADJUSTED_HRLY_VALUE AS UNADJ_NOX,
-		NOXC_MHV.APPLICABLE_BIAS_ADJ_FACTOR AS CALC_NOX_BAF,
-		NOXC_MHV.ADJUSTED_HRLY_VALUE AS RPT_ADJ_NOX,
-		NOXC_MHV.CALC_ADJUSTED_HRLY_VALUE AS ADJ_NOX_USED,
-		NOXC_MHV.MODC_CD AS NOX_MODC,
-		NOXC_MHV.PCT_AVAILABLE AS NOX_PMA,
-		FLOW_MHV.UNADJUSTED_HRLY_VALUE AS UNADJ_FLOW,
-		FLOW_MHV.APPLICABLE_BIAS_ADJ_FACTOR AS CALC_FLOW_BAF,
-		FLOW_MHV.ADJUSTED_HRLY_VALUE AS RPT_ADJ_FLOW,
-		FLOW_MHV.CALC_ADJUSTED_HRLY_VALUE AS ADJ_FLOW_USED,
-		FLOW_MHV.MODC_CD AS FLOW_MODC,
-		FLOW_MHV.PCT_AVAILABLE AS FLOW_PMA,
-		DHV.CALC_PCT_MOISTURE AS PCT_H2O_USED, 
+		hod.OP_TIME, 
+		hod.HR_LOAD AS UNIT_LOAD, 
+		hod.LOAD_UOM_CD AS LOAD_UOM, 
+		mhv.NOXC_UNADJUSTED_HRLY_VALUE AS UNADJ_NOX,
+		mhv.NOXC_APPLICABLE_BIAS_ADJ_FACTOR AS CALC_NOX_BAF,
+		mhv.NOXC_ADJUSTED_HRLY_VALUE AS RPT_ADJ_NOX,
+		mhv.NOXC_CALC_ADJUSTED_HRLY_VALUE AS ADJ_NOX_USED,
+		mhv.NOXC_MODC_CD AS NOX_MODC,
+		mhv.NOXC_PCT_AVAILABLE AS NOX_PMA,
+		mhv.FLOW_UNADJUSTED_HRLY_VALUE AS UNADJ_FLOW,
+		mhv.FLOW_APPLICABLE_BIAS_ADJ_FACTOR AS CALC_FLOW_BAF,
+		mhv.FLOW_ADJUSTED_HRLY_VALUE AS RPT_ADJ_FLOW,
+		mhv.FLOW_CALC_ADJUSTED_HRLY_VALUE AS ADJ_FLOW_USED,
+		mhv.FLOW_MODC_CD AS FLOW_MODC,
+		mhv.FLOW_PCT_AVAILABLE AS FLOW_PMA,
+		dhv.CALC_PCT_MOISTURE AS PCT_H2O_USED, 
```

Comment:
NOTE #1: All the `DHV.` aliases will need to be `dhv.NOX_` since the original DHV alias joined on the NOX parameter...
```
INNER JOIN camdecmps.DERIVED_HRLY_VALUE  DHV 
		ON DHV.HOUR_ID = HOD.HOUR_ID AND DHV.PARAMETER_CD = 'NOX'
```